scripts/modules/b2b-account/child-account.js

```diff
@@ -0,0 +1,156 @@
+define([
```

Comment:
@hrishid , Format this file in visual studio if possible.

Reply:
FIxed